lua/apg/modules/notification.lua

```diff
@@ -59,19 +64,14 @@ function APG.userNotification(msg, targets, notificationLevel, log) -- The most
 						end
 					end
 				else
-					for _, ply in next, player.GetHumans() do
-						if not IsValid(ply) then continue end
-						if not ply:IsAdmin() then continue end
-						table.insert(new_targets, ply)
-					end
+					if not IsValid(ply) then continue end
+					if not ply:IsAdmin() then continue end
```

Comment:
That ulx stuff feels kinda weird, but ok.

Reply:
In the future, I'll add a module for it.